plugin/core/test_session.py

```diff
@@ -89,39 +89,62 @@ def test_can_create_session(self):
 
     def test_can_get_started_session(self):
         project_path = "/"
-        created_callback = unittest.mock.Mock()
+        post_initialize_callbacak = unittest.mock.Mock()
```

Comment:
small typo: "callbacak"

Reply:
Hmm, does this test not run then?

Reply:
Never mind, I mindlessly used CTRL+D to change that variable name.